src/AdsImporter/Database/MongoMigration.php

```diff
@@ -173,8 +174,18 @@ public function addTransaction(ArrayableInterface $transaction): void
     {
         $document = $transaction->toArray();
 
+        if ($transaction instanceof LogAccountTransaction) {
+            $networkAccount = $document['networkAccount'];
+
+            $networkAccount['localChange'] = $this->createMongoDate($networkAccount['localChange']);
+            $networkAccount['remoteChange'] = $this->createMongoDate($networkAccount['remoteChange']);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "remoteChange" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)